src/components/Form/ManageUsers.jsx

```diff
@@ -37,7 +37,7 @@ const ManageUsers = () => {
     };
 
     if (!userObject.username) {
-      runNotification(`Operation failed. Reason: No username provided`, 5, state, dispatch);
+      runNotification('Operation failed. Reason: No username provided', 5, state, dispatch);
```

Comment:
use single quotes if it's a string literal. Only use backticks if the string is interpolated.

Reply:
Good catch. We can just revert these back to strings instead of string literals.